src/client/resources/assets/snapper/lang/pt_br.json

```diff
@@ -0,0 +1,42 @@
+{
+  "key.snapper.panorama": "Faça Capturas de Tela panorâmicas",
+  "key.snapper.huge": "Faça Captura de Tela enorme",
+  "key.snapper.screenshot_menu": "Exibir Capturas de Tela",
+  "key.snapper.recent": "Abrir a última Captura de Tela",
+  "menu.snapper.screenshotmenu": "Capturas de Tela",
+  "menu.snapper.viewermenu": "Visualizador",
+  "menu.snapper.panorama": "Exibir Panorama",
+  "button.snapper.folder": "Abrir Pasta",
+  "button.snapper.panorama.tooltip": "Visualizar o panorama atual armazenado.",
+  "button.snapper.screenshots": "Exibir Capturas de Tela",
+  "button.snapper.delete": "Excluir",
+  "button.snapper.open": "Abrir Imagem",
+  "button.snapper.view": "Exibir Imagem",
+  "button.snapper.copy": "Copiar",
+  "button.snapper.rename": "Renomear",
+  "text.snapper.created": "Criado",
+  "text.snapper.generic": "Captura de Tela",
+  "text.snapper.loading": "Carregando Capturas de Tela",
+  "text.snapper.empty": "Faça uma Captura de Tela no jogo para vê-la aqui!",
+  "text.snapper.panorama_encourage": "Faça um panorama no jogo para vê-lo aqui!",
+  "text.snapper.panorama_success": "Panorama salvo. Visualize pressionando %s",
+  "text.snapper.panorama_failure_iris": "Devido a um problema no Iris, os panoramas não podem ser tirados enquanto ele estiver instalado.",
+  "text.snapper.delete_question": "Tem certeza de que deseja excluir essa Captura de Tela?",
+  "text.snapper.delete_warning": "'%s' será perdido para sempre! (é muito tempo!)",
+  "text.snapper.rename": "Renomear Captura de Tela",
+  "text.snapper.rename_input": "Digite o novo nome aqui",
+  "text.snapper.rename_invalid": "Novo nome para a Captura de Tela inválida",
+  "text.snapper.rename_invalid_png": "O nome deve terminar com '.png'",
+  "text.snapper.screenshot_instructions": "%s. Visualizar pressionando %s",
+  "text.snapper.screenshot_instructions_copy": "%s. Copiado automaticamente para a área de transferência.",
+  "text.snapper.screenshot_failure_open": "Faça uma Captura de Tela para visualizá-la!",
+  "panorama.snapper.failure": "Não foi possível salvar o panorama: %s",
+  "panorama.snapper.success": "Captura de Tela salva como %s",
+  "config.snapper.snapper.title": "Configurações do Snapper",
+  "config.snapper.snapper.copyTakenScreenshot": "Automática-Área de transferência",
+  "config.snapper.snapper.copyTakenScreenshot.tooltip": "Copie a Captura de Tela para a área de transferência quando tirada",
+  "config.snapper.snapper.showSnapperTitleScreen": "Botão da Tela de Título",
```

Comment:
"Tela de Título" is the direct translation of Title Screen. While the translation correct, the game actually calls the title screen "Menu principal" when you're using the Brazilian translations.